packages/app/features/Send/InformationInput/components/AmountInput.tsx

```diff
@@ -0,0 +1,121 @@
+import { type FC, useState } from 'react';
+import { StyleSheet } from 'react-native';
+import { UnknownObject } from '@walless/core';
+import { Button, Input, Text, View } from '@walless/gui';
+import { useSnapshot } from 'valtio';
+
+import {
+	transactionActions,
+	transactionContext,
+} from '../../../../state/transaction';
+
+export const AmountInput: FC = () => {
+	const { token, amount } = useSnapshot(transactionContext);
+	const [errorText, setErrorText] = useState<string>();
+	const [renderBalance, setRenderBalance] = useState(false);
+
+	let balance = -1;
+	if (token) {
+		balance = parseFloat(token.account.balance) / 10 ** token.account.decimals;
```

Comment:
will accept this for now, but these kind of computations must be done with bn.js instead